# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
# Commit: b99b71e755e4
# Full Hash: b99b71e755e470a5dd152a5d6313f3002b2b9ba9
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-12-18 09:43:08
# Description:
#   Bug 1744081 - Move VideoConduit::Shutdown asyncness ownership into VideoConduit. r=bryce,ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D133933
# ==============================================================================

diff -r f69c775a1024 -r b99b71e755e4 media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
--- a/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Fri Dec 17 20:29:15 2021 +0000
+++ b/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Fri Dec 17 20:34:18 2021 +0000
@@ -328,7 +328,7 @@
       audio_pipeline_->Shutdown();
     }
     if (audio_conduit_) {
-      audio_conduit_->Shutdown();
+      Unused << WaitFor(audio_conduit_->Shutdown());
     }
     if (call_) {
       call_->Destroy();